src/usecase/user_test.go

```diff
@@ -0,0 +1,61 @@
+package usecase_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/NaokiYazawa/clean-architecture-go/domain/model"
+	mock "github.com/NaokiYazawa/clean-architecture-go/domain/repository/mock"
+	"github.com/NaokiYazawa/clean-architecture-go/usecase"
+
+	"github.com/golang/mock/gomock"
+)
+
+func Test_userUsecase_Create(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	type fields struct {
+		userRepository *mock.MockUserRepository
+	}
+	type args struct {
+		name string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		prepare func(f *fields)
+		want    *model.User
+		wantErr bool
```

Comment:
返り値が複数ある場合は`expected`みたいにまとめる方が、毎回`want/expected`とかプレフィックスつけなくていいから僕は好みかな！
```suggestion
		type expected {
		  user *model.User
		  err error
		}
		
		tests : = []strust{
		  expected expected
		}
```

Reply:
承知しました。
ありがとうございます。